[Optimization engine] Improve Storage Account access and interactive scripts security (#991)
microsoft/finops-toolkit: docs/_resources/changelog.md

>
> 1. Published initial guidance for FinOps best practices. For the initial commit, this page contains the Azure Resource Graph (ARG) queries used by the Cost Optimization workbook.

🔍 Optimization engine
{: .fs-5 .fw-500 .mt-4 mb-0 }

> ✏️ Changed:
>

> > 1. Improved Storage Account access security, replacing key-based authentication by Entra ID authentication.

Reviewer: ```suggestion
> 1. Replaced storage account key-based authentication with Entra ID authentication for improved security.
```
